filters/ELMFilter.cpp

```diff
@@ -66,43 +66,37 @@ void ELMFilter::addArgs(ProgramArgs& args)
     args.add("cell", "Cell size", m_cell, 10.0);
     args.add("class", "Class to use for noise points", m_class, uint8_t(7));
     args.add("threshold", "Threshold value", m_threshold, 1.0);
-}
+    args.add("maximum", "Compute Extended Local Maximum too", m_maximum, false);
```

Comment:
I think this a confusing name.  Perhaps some enumeration would be more clear?  Noise::Low, Noise::High, Noise::Both or some such?  Default would be Noise::High.